hyperon_das_atomdb/adapters/server_db.py

```diff
@@ -0,0 +1,277 @@
+import json
+from typing import Any, Dict, List, Optional, Tuple
+
+from requests import Session, exceptions
+
+from hyperon_das_atomdb.exceptions import (
+    ConnectionServerException,
+    NodeDoesNotExistException,
+)
+from hyperon_das_atomdb.i_database import IAtomDB
+from hyperon_das_atomdb.utils.decorators import retry
+
+
+class ServerDB(IAtomDB):
+    """A concrete implementation using servers databases.
+    AwsLambda and OpenFaas"""
+
+    def __repr__(self) -> str:
+        return "<Atom database Server>"  # pragma no cover
+
+    def __init__(
+        self,
+        database_name: str = 'das',
+        ip_address: str = None,
+        port: Optional[str] = None,
+    ) -> None:
+        if not ip_address:
+            raise ConnectionServerException(
+                message='You must send ip_address parameter'
+            )
+        self.database_name = database_name
+        self.ip = ip_address
+        self.port = port if port else '8080'
+        self.openfaas_uri = f'http://{self.ip}:{self.port}/function/atomdb'
+        self.aws_lambda_uri = f'http://{self.ip}/prod/atomdb'
+        self.session = Session()
```

Comment:
`self.session` seems to be a misleading name since I don't see a section being kept open. But perhaps I'm missing something.

Reply:
Sure. I was using Session because i was sending the credentials in a header, but it's no necessary anymore and I forgot to remove it. i'll remove it